### PR TITLE
CLN: D208 Docstring is over-indented

### PR DESCRIPTION
**Pull request by simonjayhawkins**

### CODE REVIEW

File: pandas/core/computation/pytables.py [hunk 1]
```diff
@@ -601,8 +601,7 @@ def __init__(self, value, converted, kind: str):
         self.kind = kind
 
     def tostring(self, encoding) -> str:
-        """ quote the string if not encoded
-            else encode and return """
+        """ quote the string if not encoded else encode and return """
```
Comment (jbrockmendel) [inline pandas/core/computation/pytables.py]: are the leading/trailing spaces policy?

**Comment by jbrockmendel** [comment]
LGTM

File: pandas/core/computation/pytables.py (hunk 1 above)
Comment (simonjayhawkins) [reply pandas/core/computation/pytables.py]: not according to PEP257, but that seems to popular in our codebase. There is a codecheck in pydocstyle for this, D210. This is for one-line docstrings and we seem to moving away from those, see #31162, #31462 and others.

I was wandering whether to enforce D210 yet. May be able to find a checker to find one-liners.

Comment (WillAyd) [reply pandas/core/computation/pytables.py]: I think this would fail the numpy docstring check as well were this exposed as part of the API

**Review by WillAyd — APPROVED** [review]

**Pull request merged**